### PR TITLE
show ws summary as it loads

### PR DESCRIPTION
**Pull request by zarsky-broad**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/common/components.cljs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 (react/defc Spinner
   {:render
    (fn [{:keys [props]}]
-     [:span {:style (merge {:margin "1em" :whiteSpace "nowrap"} (:style props))}
+     [:span {:style (merge {:margin "1em" :whiteSpace "nowrap" :display "inline-block"} (:style props))}
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/components.cljs]: Why is this inline-block necessary?

**Review by MattPutnam — COMMENTED** [review]
This doesn't fully address the ticket. We need to make each subcomponent that requires AJAX data own those calls, and display their own spinners and error messages. Right now, this will pop in data and swallow any errors that occur.

File: src/cljs/main/broadfcui/common/components.cljs (hunk 1 above)
Comment (zarsky-broad) [reply src/cljs/main/broadfcui/common/components.cljs]: Previously, with Spinner as an inline element, margins weren't rendered.

File: src/cljs/main/broadfcui/net.cljs [hunk 2]
```diff
@@ -36,4 +36,3 @@
           (rephrase-error ajax-response)
           (:message parsed-response))))
      :else (render-success))))
-
```
Comment (dmohs) [inline src/cljs/main/broadfcui/net.cljs:39]: Seems like you should revert this file.

**Review by dmohs — APPROVED** [review]

File: src/cljs/main/broadfcui/net.cljs (hunk 2 above)
Comment (zarsky-broad) [reply src/cljs/main/broadfcui/net.cljs:39]: Your call. It had two blank lines at the end, so I removed one.

**Comment by davidangb** [comment]
(assigning myself so I can more easily watch this PR)

**Comment by zarsky-broad** [comment]
@dmohs @MattPutnam Could I get a re-review?

**Comment by MattPutnam** [comment]
👍

**Comment by zarsky-broad** [comment]
@ahaessly I guess I hit you up for a TL thumb?

**Review by ahaessly — APPROVED** [review]
👍

**Pull request merged**